src/unified_recon/rule_0/main.py

```
@@ -0,0 +1,374 @@
+            stats = process_exchange(exchange_name, exchange_trader_trades, exchange_exchange_trades, args.show_details)
+            
+            if stats and stats["total_discrepancies"] > 0:
+                total_discrepancies += stats["total_discrepancies"]
+        
+        # Return appropriate exit code
+        if total_discrepancies > 0:
+            sys.exit(1)  # Discrepancies found
+        else:
+            sys.exit(0)  # All matched
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)